lib/oneview-sdk/resource/api500/c7000/storage_pool.rb

```diff
@@ -96,6 +96,13 @@ def request_refresh
           update
           refresh
         end
+
+        # Sets the storage system
+        # @param [OneviewSDK::StorageSystem] storage_system
+        def set_storage_system(storage_system)
+          raise IncompleteResource, 'Please set the storage system\'s uri attribute!' unless storage_system['uri']
+          set('storageSystemUri', storage_system['uri'])
```

Comment:
So, let me see if I understand. 
This method makes it so I can call:
```ruby
set_storage_system(my_storage_system)
```
with that storage system having an uri in it, instead of:
```ruby
my_storage_pool['storageSystemUri'] = my_storage_system['uri']
```
?
If this is all, I'm not entirely sure why do we need this. 
Perhaps if it uses `retrieve!` then it makes some sense, since you can just pass in an instance of storage system with a name or something.

Can someone share some light here?

Reply:
Ok... I've just seen in the issue that we do have this method for earlier API versions. 
Again, looks... really unecessary.
But sure, if we have it already, I don't see why not add it here as well. LGTM.

Reply:
It's the Ruby SDK Object Oriented standard.

However, you can do a `retrieve!` inside the method, it's more efficient and meaningful.

Reply:
Yeah, for good of our code, I'm fixing it! =D \o/